keyboards/converter/xt_usb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include "xt_usb.h"
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* IBM XT keyboard layout
+     * ,-------.  ,--------------------------------------------------------------------------.
+     * | F1| F2|  |Esc|  1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|  BS  |NumLck |ScrLck |
+     * |-------|  |--------------------------------------------------------------------------|
+     * | F3| F4|  | Tab |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ] |   |  7|  8|  9|  -|
+     * |-------|  |------------------------------------------------------|Ent|---------------|
+     * | F5| F6|  | Ctrl |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|  `|   |  4|  5|  6|   |
+     * |-------|  |----------------------------------------------------------------------|   |
+     * | F7| F8|  |Shif|  \|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift|PrS|  1|  2|  3|  +|
+     * |-------|  |----------------------------------------------------------------------|   |
+     * | F9|F10|  |  Alt  |               Space                  |CapsLck|   0   |   .   |   |
+     * `-------'  `--------------------------------------------------------------------------'
+     */
+    KEYMAP_XT( \
```

Comment:
Please replace all of the `KEYMAP_XT` with `LAYOUT` (or `LAYOUT_xt`, note the casing)